internal/common/db.go

```diff
@@ -191,6 +218,9 @@ type InfraEnv struct {
 	// Json formatted string containing internal overrides for the default ignition config.
 	// This is used for adding ironic ignition config to the assisted ignition config
 	InternalIgnitionConfigOverride string `json:"internal_ignition_config_override,omitempty"`
+
+	// A JSON blob in which holds a mirror registry configurations if set
+	MirrorRegistryConfiguration []uint8 `gorm:"type:BYTEA"`
```

Comment:
Why byte and not just serialized json?

Reply:
No reason actually, I'll change it to `serialized json` for consistency